cli/Cargo.toml

```diff
@@ -33,7 +33,7 @@ byteorder = "1.3.4"
 clap = "2.33.0"
 dirs = "2.0.2"
 dlopen = "0.1.8"
-dprint-plugin-typescript = "0.9.6"
+dprint-plugin-typescript = "0.9.9"
```

Comment:
@bartlomieju maybe update to 0.9.10 while you're here? I just fixed this issue with trailing commas: https://github.com/dsherret/dprint/issues/166 (release should be up in a few minutes)

Reply:
Sure